src/asm-manual.adoc

```diff
@@ -1079,6 +1079,64 @@ srli rd, rd, XLEN - 32
+|vfabs.v vd, vs               | vfsgnjx.vv vd, vs, vs                                         | Floating-point vector absolute value |
+|vmclr.m vd                   | vmxor.mm vd, vd, vd                                           | Vector clear mask register |
+|vmfge.vv vd, va, vb, vm      | vmfle.vv vd, vb, va, vm                                       | Vector Floating-point >=|
+|vmfgt.vv vd, va, vb, vm      | vmflt.vv vd, vb, va, vm                                       | Vector Floating-point >|
+|vmmv.m vd, vs                | vmand.mm vd, vs, vs                                           | Vector copy mask register |
+|vmnot.m vd, vs               | vmnand.mm vd, vs, vs                                          | Vector invert mask bits|
+|vmset.m vd                   | vmxnor.mm vd, vd, vd                                          | Vector set all mask bits|
+|vmsge.vi vd, va, i, vm       | vmsgt.vi vd, va, i-1, vm                                      | Vector >= Immediate|
+|vmsgeu.vi vd, va, i, vm      | vmsgtu.vi vd, va, i-1, vm                                     | Vector >= Immediate, unsigned|
+|vmsge.vv vd, va, vb, vm      | vmsle.vv vd, vb, va, vm                                       | Vector >= Vector|
+|vmsgeu.vv vd, va, vb, vm     | vmsleu.vv vd, vb, va, vm                                      | Vector >= Vector, unsigned |
+|vmsgt.vv vd, va, vb, vm      | vmslt.vv vd, vb, va, vm                                       | Vector > Vector|
+|vmsgtu.vv vd, va, vb, vm     | vmsltu.vv vd, vb, va, vm                                      | Vector > Vector, unsigned|
+|vmslt.vi vd, va, i, vm       | vmsle.vi vd, va, i-1, vm                                      | Vector < immediate|
+|vmsltu.vi vd, va, i, vm      | vmsleu.vi vd, va, i-1, vm                                     | Vector < immediate, unsigned |
+|vneg.v vd,vs                 | vrsub.vx vd,vs,x0                                             | Vector negate |
+|vnot.v vd,vs,vm              | vxor.vi vd, vs, -1, vm                                        | Vector not |
+|vncvt.x.x.w vd,vs,vm         | vnsrl.wx vd,vs,x0,vm                                          | Vector narrow convert element |
+|vwcvt.x.x.v vd,vs,vm         | vwadd.vx vd,vs,x0,vm                                          | Vector widen convert, integer-integer|
+|vwcvtu.x.x.v vd,vs,vm        | vwaddu.vx vd,vs,x0,vm                                         | Vector widen convert, integer-integer, unsigned|
+|vl1r.v v3, x0                | vl1re8.v v3, x0                                               | Equal to vl1re8.v |
+|vl2r.v v2,x0                 | vl2re8.v v2, x0                                               | Equal to vl2re8.v |
+|vl4r.v v4,x0                 | vl4re8.v v4, x0                                               | Equal to vl4re8.v |
+|vl8r.v v8,x0                 | vl8re8.v v8, x0                                               | Equal to vl8re8.v |
+
+|vmsge{u}.vx vd, va, x|
```

Comment:
This form isn't implemented in the assembler because it's impossible to know if `x>minimum` without doing more analysis than an assembler is capable of.

Reply:
Should be fixed now, thanks!

Reply:
My bad, I messed up when commiting, now should be fixed. Sorry for the mess up.